stacker/commands/stacker/base.py

```diff
@@ -101,6 +117,11 @@ def run(self, options, **kwargs):
     def configure(self, options, **kwargs):
         self.logger_type = setup_logging(options.verbose, options.interactive)
 
+    def semaphore(self, options):
+        if options.interactive:
+            return threading.Semaphore(1)
```

Comment:
Currently, dropping down to interactive mode removes parallelism, which isn't actually necessary. It would be better to add locks around the change set prompts.